src/styles/login/login.js

```diff
@@ -22,7 +22,10 @@ const textStyles: { [string]: TextStyleProp } = StyleSheet.create( {
   },
   input: {
     color: "#000000",
-    backgroundColor: "#ffffff"
+    backgroundColor: "#ffffff",
```

Comment:
Pondering of the week and something I wish I had thought of earlier: should we be using a component library like [react-native-elements](https://reactnativeelements.com/) for placeholders while we're waiting for final designs? That might make it easier to navigate the app, and ideally we can find a component library that will allow us to customize styling.

Reply:
Yes please. No matter what the final design is, a theming system is a good idea for consistency.